src/styles.scss

```diff
@@ -424,3 +424,7 @@ span, div, label, p {
   max-width: 1400px;
   margin:auto;
 }
+
+.align {
```

Comment:
no need to define a new class to apply this style(text-align: center), class  .center   in style.css do that